src/components/filter-box/extended-filter-box.tsx

```diff
@@ -4,6 +4,16 @@ import ReactFilterBox from "@netdata/react-filter-box"
 // So we extend base component and update its parser manually
 // to enable autosuggestions for dynamic data
 export class ExtendedFilterBox extends ReactFilterBox {
+  componentDidMount() {
+    if (super.componentDidMount) {
+      super.componentDidMount()
+    }
+    if (this.props.query) {
+      const result = this.parser.parse(this.props.query)
+      if (this.props.onParseOk) this.props.onParseOk(result)
```

Comment:
We should better use bracketed conditionals if possible. 
Both for maintainability and readability...
https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces

Reply:
The problem with readability lies on the `if..else`. One liner if is fine afaic.